RadixWallet/SARGON/NewSargonTypes/AssetTransferRepresentation.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+// MARK: - AssetsTransfersTransactionPrototype
+// NOT FINAL, just a sketch, probably wrong...
+public struct AssetsTransfersTransactionPrototype {
```

Comment:
Non final representation of the state created by user in UI before she presses "Continue" from AssetTransfer